### PR TITLE
Story2 splash screen

### PR DESCRIPTION
**Pull request by C0deJack**
This story incorporates the splash screen and instructions pop up.

### CODE REVIEW

File: css/splash.css [hunk 1]
```diff
@@ -0,0 +1,106 @@
+.splash {
+    position: relative;
```
Comment (charliecog) [inline css/splash.css:2]: unused

File: css/splash.css [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    width: 600px;
+    margin: 100px auto;
+}
+
+.splash .instructionsBtn {
+    display: block;
+    width: 600px;
+    height: 50px;
+    margin: 50px auto;
+    text-align: center;
```
Comment (charliecog) [inline css/splash.css]: unused

File: css/splash.css [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    font-family: 'Press Start 2P', cursive;
+    border-radius: 5px;
+    color: #ffffff;
+    text-shadow: 2px 2px #000000, 2px -2px #000000, -2px -2px #000000, -2px 2px #000000;
+    background-color: #dddddd;
+}
+
+.splash .instructionsBtn:hover {
+   cursor: pointer;
+   background-color: #dddddd;
```
Comment (charliecog) [inline css/splash.css]: unused

File: css/splash.css [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+.splash .gameLevel img {
+    display: block;
+    width: 350px;
+    margin: 0 auto;
+    border: 1px solid #000000;
+    border-radius: 5px;
+}
+
+.splash .gameLevel h2  {
+    width: 350px;
```
Comment (charliecog) [inline css/splash.css]: unused

File: css/splash.css [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+    left: 100px;
+    top: 100px;
+    border-radius: 5px;
+    background-color: #b3ff0f;
+}
+
+.instructions button {
+    font-family: 'Press Start 2P', cursive;
+    font-size: 1.6rem;
+    width: 500px;
```
Comment (charliecog) [inline css/splash.css]: unused

File: css/splash.css [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+    text-align: center;
```
Comment (charliecog) [inline css/splash.css]: unused

File: index.html [hunk 7]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
```
Comment (charliecog) [inline index.html]: missing a lang attribute

File: index.html [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+<head>
+    <link rel="stylesheet" type="text/css" href="css/normalize.css">
+    <link rel="stylesheet" type="text/css" href="css/styles.css">
+    <link rel="stylesheet" type="text/css" href="css/splash.css">
+    <link href="https://fonts.googleapis.com/css?family=Press+Start+2P" rel="stylesheet">
+</head>
+<body>
+    <div class="container">
```
Comment (charliecog) [inline index.html]: could be a ```<main>```

File: index.html [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+        <div class="splash">
+            <img src="assets/gameTitle.png" alt="Mayrio Kart">
+            <button id="pullInstructions"  class="instructionsBtn">
+                Instructions
+            </button>
+            <div class=" gameLevel">
```
Comment (charliecog) [inline index.html]: whitespace

File: index.html [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+            <div class="instructions" id="instructions"></div>
```
Comment (charliecog) [inline index.html]: having a class and an id on the same element with the same name isnt very verbose

File: instructions.js [hunk 11]
```diff
@@ -0,0 +1,10 @@
+
```
Comment (charliecog) [inline instructions.js]: whitespace

**Review by charliecog — CHANGES_REQUESTED** [review]
Looks good - Couple of things to do.
- other changes required to do with your directory structure: .js files should be in a js folder and any .html
files that aren't index.html should be in includes folder as agreed.

File: css/splash.css (hunk 1 above)
Comment (C0deJack) [reply css/splash.css:2]: this is being used by the instructions pop up div

File: css/splash.css (hunk 2 above)
Comment (C0deJack) [reply css/splash.css]: text-align: center; removed

File: css/splash.css (hunk 3 above)
Comment (C0deJack) [reply css/splash.css]: this is being used to colour the instructions button background to grey

File: css/splash.css (hunk 4 above)
Comment (C0deJack) [reply css/splash.css]: removed

File: index.html (hunk 7 above)
Comment (C0deJack) [reply index.html]: added

File: index.html (hunk 8 above)
Comment (C0deJack) [reply index.html]: good shout thanks

File: index.html (hunk 9 above)
Comment (C0deJack) [reply index.html]: done

File: index.html (hunk 10 above)
Comment (C0deJack) [reply index.html]: Removed id attribute and changed instructions.js file to point to the class rather than the id

File: instructions.js (hunk 11 above)
Comment (C0deJack) [reply instructions.js]: done

File: css/splash.css (hunk 1 above)
Comment (charliecog) [reply css/splash.css:2]: sounds good

File: css/splash.css (hunk 3 above)
Comment (charliecog) [reply css/splash.css]: as discussed in person, this is a duplication of line 21

File: css/splash.css (hunk 5 above)
Comment (charliecog) [reply css/splash.css]: ?

File: css/splash.css (hunk 6 above)
Comment (charliecog) [reply css/splash.css]: ?

**Review by charliecog — CHANGES_REQUESTED** [review]
couple of last things

File: css/splash.css (hunk 3 above)
Comment (C0deJack) [reply css/splash.css]: done

File: css/splash.css (hunk 5 above)
Comment (C0deJack) [reply css/splash.css]: removed -sorry didn't update comment

Comment (C0deJack) [reply css/splash.css]: or css... doh

**Review by charliecog — APPROVED** [review]

**Pull request merged**